src/webgpu/api/validation/encoding/beginComputePass.spec.ts

```diff
@@ -0,0 +1,57 @@
+export const description = `
+Tests for validation in beginComputePass and GPUComputePassDescriptor as its optional descriptor.
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { ValidationTest } from '../validation_test.js';
+
+export const g = makeTestGroup(ValidationTest);
+
+g.test('compute_pass_descriptor_location')
```

Comment:
since the file is called `beginComputePass` it's not that necessary to also say `compute_pass_descriptor`. Shorter is nice for navigability as long as it's still easy to understand.

I would call this "timestamp_writes_location" to match the equivalent `createRenderPass` test, or `timestampWrites,same_location` to use the name of the descriptor field and a slightly more descriptive name for the test.

Reply:
Ah, I see. Then, let me use `timestampWrites,same_location` for the test. Thanks.